### ISSUE TITLE
Missing File's ?

### ISSUE THREAD

**Issue by kefkevm** [djgreeb/Sampler-DJS-500#6]
Hi,

It looks like some files are missing?
See the screenshot's i took.

![File missing](https://user-images.githubusercontent.com/57961226/221375574-19a3df11-6b3a-4da5-b218-c2c56dab435d.JPG)
![missing files](https://user-images.githubusercontent.com/57961226/221375578-05be3ffb-4826-4ccc-9b7d-4d89c8f09efb.JPG)

**Comment by djgreeb** [comment]
https://drive.google.com/file/d/18DZ3agTO7aAusT0o3a3R0Xqb5H65mn98/view?usp=sharing

**Comment by kefkevm** [comment]
Thanks a Lot, 

I think it worked, but don't  have te full license for compiling.. 
do you have maybe a compiled .hex version as wel? 

Thanks so far :)

![Almost](https://user-images.githubusercontent.com/57961226/222230927-f7e64da2-4627-44f4-b3ba-dcc2157ef8f0.JPG)

**Comment by ultraw3ird** [comment]
Hi, any future plans for releasing compiled fw? Unofrtunately without licensed Keil there is no way to get fw compiled due to size of project :/

**Comment by kefkevm** [comment]
you have to upload the f7.hex file to the board. that's the compiled version. 

You can find it here:
Firmware/Sampler/HEX

**Comment by ultraw3ird** [comment]
Got it, have you managed get the fw running ? mine is stuck in reset loop after uploading, probably some hw mod of board needed ?

**Comment by kefkevm** [comment]
yes, you have to modify the backlight. remove R85, detach input from D8(PK3) and connect driver input to L2 (PF9). L2 = A2 in het connection pins. see the picture below.


![Backlight](https://github.com/djgreeb/Sampler-DJS-500/assets/57961226/faba6107-db68-48df-a3f8-292eb1d44beb)

**Comment by ultraw3ird** [comment]
Works flawlessly. Only for info am I blind as usual and is this info written somewhere in available docs or you got this determined from CPU pinout and code ? Thanks you again for your support :) Also have you progressed with building hw around ? I am missing few bits and pieces to be wired. Could we connect directly to discuss this to not spam here and there ?

**Comment by kefkevm** [comment]
yes, you can connect directly to me. send me a DM https://www.instagram.com/feest_dj_kevin/

**Comment by mattpstv** [comment]
> Works flawlessly. Only for info am I blind as usual and is this info written somewhere in available docs or you got this determined from CPU pinout and code ? 

@ultraw3ird did you ever go any further with this conversation? I am also blind and curious how @kefkevm came to this solution.

 I did manage to get the firmware loaded (on two different stm32f746g-disco boards), and I see the splash screen, then an alert about no SD card found (unless I have an SD inserted), and then the main interface screen showing PLAYER 3 and "sampler preset 004" - but that's as far as I get. the track info boxes, BPM, and PRO DJ LINK remain unpopulated. I'm going by this video for what I assume us supposed to happen if hex file is uploaded correctly: https://youtu.be/rUySWEsYK1w

im hesitant to start breadboarding the pads, encoder and pots etc. until I know that I at least have a functional board.
